app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectPresenter.kt

```diff
@@ -176,14 +181,21 @@ class DeviceConnectPresenter(deviceActivity: DeviceActivity, manager: WifiManage
     }
 
     override fun makeWifiRequest(ssid: String, password: String) {
-        Timber.d("In here")
+        Timber.d("In here : WIFI REQUEST")
+        deviceConnectView?.showProgress(utilModel.getString(R.string.connecting_device))
         deviceModel.sendWifiCredentials(ssid, password, this@DeviceConnectPresenter)
     }
 
     override fun makeConfigRequest() {
+        Timber.d("In here : CONFIG REQUEST")
+        deviceConnectView?.showProgress(utilModel.getString(R.string.connecting_device))
+        deviceModel.setConfiguration("google", "google", "y", "n", this@DeviceConnectPresenter)
     }
 
     override fun makeAuthRequest() {
+        Timber.d("In here : AUTH REQUEST")
+        deviceConnectView?.showProgress(utilModel.getString(R.string.connecting_device))
+        deviceModel.sendAuthCredentials("y", "mohitkumar2k15@dtu.ac.in", "batbrain", this@DeviceConnectPresenter)
```

Comment:
why hardcoded credentials?

Reply:
The PR is basic connection and in PRs earlier I have mentioned the reason for them, refer those PRs

Reply:
Sorry, can't recall specifically which one. Please link me to the PRs.